safe/gui/tools/shake_grid/shake_grid.py

```diff
@@ -821,6 +837,99 @@ def set_contour_properties(self, input_file):
 
         layer.commitChanges()
 
+    def mmi_point_sampling(self, place_layer, raster_mmi_path):
+        """Get MMI information from ShakeMap Tiff on place location.
+
+        Get the MMI value on each cities/places from the generated tiff files.
+
+        :param place_layer: Point layer generated from distance calculation
```

Comment:
We try to add a full stop at the end of each `:param` and `:return:`. Better if it can be done, but no worries ;-)

Reply:
Thanks for reminding me, I must've missed that